packages/engine/tests/units/message/all_types/integration-test.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "steps": 3,
+    "expected-output": {
+      "json-state": {
+        "1": [
+          {
+            "messages": [
+              {
+                "type": "test",
+                "to": ["1"]
```

Comment:
Are we making a conscious decision to **_not_** test that the message body is correctly present in the json_state?